addons/mrp/tests/test_order.py

```diff
@@ -4916,6 +4916,50 @@ def test_mo_without_resource_calendar(self):
         self.assertEqual(mo.state, 'done')
         self.assertEqual(mo.workorder_ids[0].duration_expected, 1440.0)
 
+    def test_plan_depending_workorder(self):
+        """
+        Checks that modifying the real duration of a workorder depending on other workorders that have already been started works.
+        """
+        # create a production with two workorders, one depending on the other
+        self.env.user.groups_id += self.env.ref('mrp.group_mrp_routings')
+        self.env.user.groups_id -= self.env.ref('mrp.group_mrp_workorder_dependencies')
+
+        self.product_depends = self.env['product.product'].create({
+            'name': 'depends',
+        })
+
+        self.bom_depends = self.env['mrp.bom'].create({
+            'product_id': self.product_6.id,
+            'product_tmpl_id': self.product_6.product_tmpl_id.id,
+            'product_uom_id': self.uom_dozen.id,
+            'ready_to_produce': 'asap',
+            'consumption': 'flexible',
+            'product_qty': 1.0,
+            'allow_operation_dependencies': True,
+            'operation_ids': [
+                (0, 0, {'name': 'Cutting Machine', 'workcenter_id': self.workcenter_1.id, 'time_cycle': 12, 'sequence': 1}),
+                (0, 0, {'name': 'Weld Machine', 'workcenter_id': self.workcenter_1.id, 'time_cycle': 18, 'sequence': 2}),
```

Comment:
```suggestion
                Command.create({'name': 'Cutting Machine', 'workcenter_id': self.workcenter_1.id, 'time_cycle': 12, 'sequence': 1}),
                Command.create({'name': 'Weld Machine', 'workcenter_id': self.workcenter_1.id, 'time_cycle': 18, 'sequence': 2}),
```
[Commands](https://www.odoo.com/documentation/18.0/developer/reference/backend/orm.html#odoo.fields.Command) are a bit easier to read. :slightly_smiling_face: